crates/node/src/interactive_cli/identity.rs

```diff
@@ -54,24 +54,21 @@ enum AliasSubcommands {
         aliases = ["rm", "del", "delete"],
     )]
     Remove {
-        /// Name of the alias to remove
         identity: Alias<PublicKey>,
-        /// The context that the identity is a member of (omit to use default context)
-        #[clap(long = "context", short = 'c')]
-        context: Option<Alias<ContextId>>,
+        /// The context that the identity is a member of (must be specified, use "default" for the default context)
+        #[clap(long, short, required = true)]
+        context: Alias<ContextId>,
```

Comment:
is this intentional? it makes sense, to prevent unintentional deletion, but just checking

If it is, you don't need `required` or the rest of the comments, since that is already the behaviour

```suggestion
        /// The context that the identity is a member of
        #[clap(long, short)]
        context: Alias<ContextId>,
```

Reply:
Yes it was intentional for unintentional deletion of `default` alias , but if it s. behaviour like that then i ll remove the required part